.mailmap

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2024, Rylie Pavlik
+# SPDX-License-Identifier: Unlicense
+
+Alec Delaney <tekktrik@gmail.com>
+Alec Delaney <tekktrik@gmail.com> <89490472+tekktrik@users.noreply.github.com>
+Alec Delaney <tekktrik@gmail.com> <tekktik@gmail.com>
+Carter Nelson <caternuson@gmail.com>
+Dylan Herrada <dylan.herrada@adafruit.com>
+Dylan Herrada <dylan.herrada@adafruit.com> <33632497+dherrada@users.noreply.github.com>
```

Comment:
Please map these to Eva as well.

Reply:
Sorry, happy to fix. Didn't realize that, my deepest apologies to Eva. I had assumed they were a couple.

Reply:
No problem. I'm not sure she knew that mailmap existed (I didn't.) :-)